dev.hdos.HDOS.json

```diff
@@ -0,0 +1,92 @@
+{
+    "id": "dev.hdos.HDOS",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "hdos",
+    "separate-locales": false,
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--env=JAVA_HOME=/app/jre",
+        "--talk-name=org.freedesktop.Notifications",
+        "--filesystem=xdg-run/app/com.discordapp.Discord:create",
+        "--filesystem=~/hdos:create",
+        "--persist=jagexcache",
+        "--filesystem=home"
+
+    ],
+    "modules": [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [
+                "/usr/lib/sdk/openjdk11/install.sh"
+            ]
+        },
+        {
+            "name": "libnotify",
+            "buildsystem": "meson",
+            "config-opts": ["-Dman=false", "-Dgtk_doc=false", "-Ddocbook_docs=disabled"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.gnome.org/GNOME/libnotify/-/archive/0.8.1/libnotify-0.8.1.tar.gz",
+                    "sha256": "7c0b252edecbf08db50d775f9e720ecc03c742fb97c25f3966a8b7a4bedf8133"
+                }
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig",
+                "/lib/debug",
+                "/lib/girepository-1.0",
+                "/share/gtk-doc",
+                "/share/gir-1.0"
+            ]
+        },
+        {
+            "name": "hdos",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D hdos /app/bin/hdos",
+                "install -D hdos-launcher.jar /app/share/hdos-launcher.jar",
+                "install -Dm644 dev.hdos.HDOS.png /app/share/icons/hicolor/128x128/apps/dev.hdos.HDOS.png",
+                "install -Dm644 dev.hdos.HDOS.desktop /app/share/applications/dev.hdos.HDOS.desktop",
+                "install -Dm644 dev.hdos.HDOS.appdata.xml /app/share/appdata/dev.hdos.HDOS.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "hdos",
+                    "commands": [
+                        "for file in $XDG_RUNTIME_DIR/app/com.discordapp.Discord/discord-ipc-*; do ln -sf $file $XDG_RUNTIME_DIR/$(basename $file); done",
+                        "Exec=env vblank_mode=0 $JAVA_HOME/bin/java -jar /app/share/hdos-launcher.jar \"$@\""
+                    ]
+                },
+                {
+                    "type": "file",
+                    "url": "https://cdn.hdos.dev/launcher/latest/hdos-launcher.jar",
```

Comment:
since this file isn't versioned, if upstream has a new version, this will break.

Reply:
I'll version it